kitsune/sumo/static/sumo/js/device-migration-wizard.js

```diff
@@ -3,6 +3,32 @@ import trackEvent from "sumo/js/analytics";
 import SwitchingDevicesWizardManager from "sumo/js/switching-devices-wizard-manager";
 import "sumo/js/form-wizard";
 
-$(document).ready(function() {
+document.addEventListener("DOMContentLoaded", function () {
   new SwitchingDevicesWizardManager(document.querySelector("#switching-devices-wizard"))
+  kbTabsInit();
 });
+
+function kbTabsInit() {
+  document.getElementById("kb-tab-all").style.display = "block";
+  let tabItems = document.querySelectorAll('[data-event-category="link click"]');
+  tabItems.forEach((item) => {
+    item.addEventListener("click", (ev) => {
+      toggleTabContent(ev);
+    });
+  });
+}
+
+function toggleTabContent(ev) {
+  let tabContent = document.getElementsByClassName("topic-list");
+  [].forEach.call(tabContent, (el) => {
```

Comment:
Would this work instead?
```suggestion
  tabContent.forEach((el) => {
```